db/migrate/20160406132213_create_flood_risk_engine_organisations.rb

```diff
@@ -8,6 +8,6 @@ def change
 
       t.timestamps null: false
     end
-    add_foreign_key :flood_risk_engine_organisations, :flood_risk_engine_contacts
+
```

Comment:
@woodpigeon  It was moved. Postgres choked as the Contact table was not present at this time,  so I moved it to the bottom of the next migration which created the Contact table